Scavenge-UW/Scavenge
# Implement add item to inventory

**How to Test**
1. As an admin, go to Pantry Inventory Management.
2. Add an item that does not exist.
![image](https://user-images.githubusercontent.com/13430641/112702339-280f5f00-8e61-11eb-87af-413f999b35a5.png)
3. Item should appear in the inventory.
![image](https://user-images.githubusercontent.com/13430641/112702345-2ba2e600-8e61-11eb-967e-a3fc5d687aa1.png)

**Change Summary**
Implements inventory item addition
